### DISCUSSION TITLE
Docker file or Image for HLS4ML

### DISCUSSION THREAD

**Discussion by holoLens8** [fastmachinelearning/hls4ml#814] [Q&A] [answered]
Hello all, 

Is there any docker image for hls4ml that works fine ?
I've been struggling in building an image using the docker file provided in the test directory. The build is always failing due to the following errors:

E: Repository 'http://security.debian.org/debian-security buster/updates InRelease' changed its 'Suite' value from 'stable' to 'oldoldstable'
E: Repository 'http://deb.debian.org/debian buster InRelease' changed its 'Suite' value from 'stable' to 'oldoldstable'
E: Repository 'http://deb.debian.org/debian buster-updates InRelease' changed its 'Suite' value from 'stable-updates' to 'oldoldstable-updates'

Thank you.

**Comment by vloncar** [comment, marked as answer]
Debian had a release a few days ago, I'm guessing the downstream miniconda project didn't update docker image that we use, so when we pull the latest version this happens. Looks like they don't update it anymore, and the new docker image is `continuumio/miniconda3` so try changing the first line of `Dockerfile` to that.

**Reply by holoLens8** [reply]
Thank you for your answer. I'll try it!
